fe/fe-core/src/main/java/org/apache/doris/nereids/rules/expression/rules/RangeInference.java

```diff
@@ -461,6 +475,9 @@ public boolean isAnd() {
 
         @Override
         public ValueDesc union(ValueDesc other) {
+            if (other instanceof EmptyValue) {
+                return other.union(this);
+            }
```

Comment:
add some comment to explain this

Reply:
for a given reference,   EmptyValue(reference) or UnknownValue(reference),  then it should always evaluate to UnknowValue(referece)

Reply:
add these comment into code please

Reply:
had add